File: wrappers/mvc/demos/Kendo.Mvc.Examples/Models/Diagram/DiagramNode.cs
using System;
using System.Collections.Generic;
namespace Kendo.Mvc.Examples.Models
{
    public class DiagramNode
    {
        public DiagramNode()
        {
        }

        public DiagramNode(string name)
        {
            Name = name;
            Items = new List<DiagramNode>();
        }

        public string Name { get; set; }

        public List<DiagramNode> Items { get; set; }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/models/DiagramNode.java
package com.kendoui.spring.models;

import java.util.ArrayList;
import java.util.List;

public class DiagramNode {
    private String name;
    private List<DiagramNode> items = new ArrayList<DiagramNode>();

    public DiagramNode(String name) {
        setName(name);
    }
    
    public List<DiagramNode> getItems() {
        return items;
    }
    
    public void setItems(List<DiagramNode> items) {
        this.items = items;
    }
    
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }}